lifemonitor/utils.py

```diff
@@ -8,6 +8,7 @@
 import requests
 
 RO_CRATE_METADATA_FILENAME = "ro-crate-metadata.jsonld"
+RO_CRATE_TEST_DEFINITION_FILENAME = "test-suite-definition.json"
```

Comment:
The metadata file is called `test-metadata.json` in the [current specs](https://github.com/crs4/life_monitor/wiki/Test-Metadata-Draft-Spec).

Reply:
Fixed by https://github.com/crs4/life_monitor/pull/32/commits/1d83f48a4df8fee2d163eff204d3d9f2f58e9e80